weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -517,13 +522,14 @@ def get_object_ref_root_val(r: refs.ObjectRef) -> typing.Any:
                 val = root_val_cache[cache_key]
             else:
                 if r.version == "latest":
-                    conds = [
-                        "name = {name: String}",
-                        "is_latest = 1",
-                    ]
-                    parameters = {
-                        "name": r.name,
-                    }
+                    raise NotFoundError("Reading refs with `latest` is not supported")
```

Comment:
This is a "feature" i recently added but we want to keep it disabled for now as the need for it represents a client issue. I forgot to comment it out on the last pr

Reply:
cool, I was going to do a pass here: https://wandb.github.io/weave/guides/tracking/objects I'll make some edits after I test stuff